frontend/src/middleware.ts

```diff
@@ -0,0 +1,15 @@
+import { NextResponse } from "next/server";
+import type { NextRequest } from "next/server";
+
+export function middleware(request: NextRequest){
+    const isLoggedIn = request.cookies.get("isLoggedIn");
+
+    if(isLoggedIn && isLoggedIn.value=="0"){
+        return Response.redirect(new URL("/login", request.nextUrl))
+    }
+}
+
+
+export const config = {
+    matcher: ["/logintest"]
```

Comment:
The login is required only for the betting system right?

Reply:
yeah, we should probably talk about how we want the login/users to function since we want to have people login to post the matches as well probably